lib/features/composer/domain/model/sending_email.dart

```diff
@@ -56,6 +56,28 @@ class SendingEmail with EquatableMixin {
     return val;
   }
 
+  factory SendingEmail.fromJson(Map<String, dynamic> json) {
+
+    return SendingEmail(
+      sendingId: json['sendingId'] as String,
+      email: Email.fromJson(jsonDecode(json['email'])),
+      emailActionType: _getEmailActionType(json['emailActionType'] as String),
+      sentMailboxId: json['sentMailboxId'] != null ? const MailboxIdNullableConverter().fromJson(json['sentMailboxId'] as String?) : null,
```

Comment:
You can write `sentMailboxId:  const MailboxIdNullableConverter().fromJson(json['sentMailboxId'] as String?)`